core/src/main/java/org/streamprocessor/core/utils/BqUtils.java

```diff
@@ -642,6 +663,6 @@ public static TableRow convertJsonToTableRow(String entity, String json) {
     }
 
     public static DateTime convertStringToDatetime(String entity, String timestamp) {
-        return (DateTime) JSON_VALUE_PARSERS.get(TypeName.DATETIME).apply(timestamp);
+        return convertToDatetime(timestamp, false);
```

Comment:
Do we always want false here or should we add the relax check here too?

Reply:
It's only used for converting the event_timestamp so I think it's fine